extensions/ql-vscode/snippets.json

```diff
@@ -0,0 +1,134 @@
+{
+  "Query Metadata": {
+    "prefix": "querymetadata",
+    "body": [
+      "/**",
+      " * @name $1",
+      " * @description $2",
+      " * @kind $3",
```

Comment:
Either add a _choice_ here, or separate this into multiple snippets, one for each metadata kind.

https://code.visualstudio.com/docs/editor/userdefinedsnippets#_choice